site/_docs_1_0/httpproxy.md

```diff
@@ -996,6 +996,45 @@ spec:
       weight: 20
 ```
 
+### TCPProxy delegation
+
+There can be at most one TCPProxy stanza per root HTTPProxy, however that TCPProxy does not need to be defined in the root HTTPProxy object.
+HTTPProxy authors can delegate the configuration of a TCPProxy to the TCPProxy configuration defined in a HTTPProxy child object.
+
+```yaml
+# httpproxy-parent-termination.yaml
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: parent
+  namespace: default
+spec:
+  virtualhost:
+    fqdn: tcp.example.com
+    tls:
+      secretName: secret
+  tcpproxy:
+    include:
+      name: child
+      namespace: app
+---
+# httpproxy-child-termination.yaml
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: child
+  namespace: app
+spec:
+` tcpproxy:
```

Comment:
Nit: extraneous backquote.

Reply:
I remember looking at that last night and thinking it was a spec of crud on my screen :)